src/components/Button.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Text, View, StyleSheet, TouchableOpacity } from 'react-native';
+
+export type ButtonProps = {
+    title: string;
+    appearance?: string;
+    onPress?: () => void;
+}
+
+export default function Button({ title, appearance, onPress }: ButtonProps) {
+  const lightStyle = appearance === 'light';
```

Comment:
you can set default value inside the props arguments:

`function Button({ title, appearance = 'light', onPress }: ButtonProps)`